api/testdata/complete-scenarios/SF85P/test2.json

```diff
@@ -7985,8 +7985,7 @@
                   "Officer": {
                     "type": "radio",
                     "props": {
-                      "name": "officer-na",
-                      "value": "NotApplicable",
+                      "value": "Enlisted",
```

Comment:
Just to clarify: the XSD error got a little confused because of the same name `Rank`, but Enlisted is not actually a rank, but since this field is free form, it's just as good as any other string.